File: frontend/client/lib/models/audio_file.dart
class AudioFile {
  final int audioId;
  final String url;
  final int? duration;
  final String format;

  AudioFile({
    required this.audioId,
    required this.url,
    this.duration,
    required this.format,
  });

  factory AudioFile.fromJson(Map<String, dynamic> json) {
    return AudioFile(
      audioId: json['audioId'],
      url: json['url'],
      duration: json['duration'],
      format: json['format'] ?? 'mp3',
    );
  }
}

File: frontend_fix/client/lib/model/AudioFile.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.Text.Json.Serialization;

namespace client.lib.model
{
    public class AudioFile
    {
        [JsonPropertyName("audioId")]
        public int AudioId { get; set; }

        [JsonPropertyName("url")]
        public string Url { get; set; } = string.Empty;

        [JsonPropertyName("duration")]
        public int? Duration { get; set; }

        [JsonPropertyName("format")]
        public string Format { get; set; } = "mp3";
    }
}
